Header header
int8 location_row
int8 location_col
float32 heading